### ISSUE TITLE
Blocks: "Task List" block doesn't allow adding more than one item

### ISSUE THREAD

**Issue by lancewillett** [Automattic/wp-calypso#39891]
Possibly a regression after this was added to the Calypso repo by CC @sirreal  and @Automattic/team-calypso, see #32058

#### Steps to reproduce
1. Starting at URL: https://wpcalypso.wordpress.com/post/[site]
2. Type `/Task` to get the Task List block
3. Try to add more than one item

#### What I expected
To hit "Enter" with the keyboard to get a second task item. (Or use visual controls such as a "plus" button.)

#### What happened instead
No visual control; "Enter" with keyboard creates a newline but not a new task.

#### Browser / OS version
Chrome latest on macOS

#### Screenshot / Video
![task-list](https://user-images.githubusercontent.com/66797/75944216-597f4c00-5e54-11ea-89de-bcfb5f3876d0.gif)

Found by @peterfabian and @mkaz during a usability session today.

CC @mtias

**Comment by sirreal** [comment]
It's unlikely that this was introduced by https://github.com/Automattic/wp-calypso/pull/32058 and I'd speculate that this was caused by updates to the Gutenberg plugin while o2 blocks are unmaintained.

**Comment by lancewillett** [comment]
Right, I linked to that as the entry point – a bit confusing why o2 blocks would be in Calypso repo. Why not in the o2 plugin? https://github.com/Automattic/o2

Next action: if the blocks don't work — and no plans to maintain and fix — please plan to remove them. Especially if other blocks do a better job at the same thing.

**Comment by sirreal** [comment]
In D39941-code I just rebuilt the sources and it seems to fix the issue. Not sure what the problem was 🤷‍♂

**Comment by lancewillett** [comment]
You rock! Will review and accept shortly.

**Comment by lancewillett** [comment]
Confirmed fixed.

![task-list-fix](https://user-images.githubusercontent.com/66797/76015042-d69fd500-5ed7-11ea-8d48-e7a47d5f22f6.gif)

UX note — for later, not related to this.

Can't use keyboard as expected — Enter for new item, and back or up for navigating the list.

**Comment by sirreal** [comment]
Fix is deployed, amigos 😉
